### PR TITLE
Fixed right number of month in the notes section

### PR DESCRIPTION
**Pull request by heron2014**
Tiny issue related to [issue202](https://github.com/FAC-GM/app/issues/202)
First PR (2016), do you mind check it?  :smile: @nelsonic

### CODE REVIEW

**Comment by nelsonic** [comment]
@heron2014 looks good.
*however* we should *also* add a test for this to confirm its working 
and given that the *purpose* of the code is not immediately *obvious* to our [*future selves*](http://blog.codinghorror.com/coding-for-violent-psychopaths) a comment would not hurt.
you can simply add a link to the issue e.g: `// see: http://git.io/vuZOK`
:wink:

**Comment by heron2014** [comment]
@nelsonic you right, tests are needed for our sanity :smile:

**Comment by nelsonic** [comment]
@heron2014 if you have time to write an assertion in an *existing* test and add the comment
with the link we can merge this in today. :+1:

**Comment by heron2014** [comment]
@nelsonic please check the added test.

File: lib/notes.js [hunk 1]
```diff
@@ -26,8 +26,8 @@ exports.register = function (server, options, next) {
           notes.company = request.payload.company;
           notes.notes = request.payload.notes;
           notes.author = request.auth.credentials.firstname;
-          var currentDate = new Date();
-          notes.createdAt = currentDate.getDate() + '-' + currentDate.getMonth() + '-' + currentDate.getFullYear();
+          var currentDate = new Date(); 
+          notes.createdAt = currentDate.getDate() + '-' + ("0" + (currentDate.getMonth() + 1)).slice(-2) + '-' + currentDate.getFullYear();
```
Comment (nelsonic) [inline lib/notes.js]: see: http://git.io/vuZOK

**Comment by nelsonic** [comment]
Nice one. :+1:

**Pull request merged**